<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="TAB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1677780</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="basic_tab">
       <property name="toolTip">
        <string/>
       </property>
       <attribute name="title">
        <string>Basic</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_ping">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>ping 스캔</string>
            </property>
            <widget class="QLineEdit" name="ping_ip">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>50</y>
               <width>231</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="lbl_stealth_ip_3">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>50</y>
               <width>41</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>IP대역</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_ping">
             <property name="geometry">
              <rect>
               <x>240</x>
               <y>110</y>
               <width>80</width>
               <height>23</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);
	border-radius: 5px;
</string>
             </property>
             <property name="text">
              <string>스캔시작</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn_allreset">
            <property name="styleSheet">
             <string notr="true">color: white;
	background-color:rgb(255, 0, 110);</string>
            </property>
            <property name="text">
             <string>All_Reset</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_syn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>170</height>
             </size>
            </property>
            <property name="title">
             <string>SYN 스캔</string>
            </property>
            <widget class="QLabel" name="lbl_stealth_ip_5">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>50</y>
               <width>21</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>IP</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_syn_ip">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>50</y>
               <width>231</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="lbl_stealth_port_3">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>90</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="syn_start_port">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>90</y>
               <width>91</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>100</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>~</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="syn_end_port">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>90</y>
               <width>91</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_syn">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>130</y>
               <width>80</width>
               <height>23</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(234, 116, 255);
	background-color: white;
	border: 2px solid rgb(234, 116, 255);
	border-radius: 5px;</string>
             </property>
             <property name="text">
              <string>스캔시작</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox_stealth">
            <property name="title">
             <string>Stealth 스캔</string>
            </property>
            <widget class="QComboBox" name="combo_stealth">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>10</y>
               <width>151</width>
               <height>22</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(230, 230, 230);</string>
             </property>
             <item>
              <property name="text">
               <string>Half Scan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Xmas Scan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fin Scan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Null Scan</string>
              </property>
             </item>
            </widget>
            <widget class="QLabel" name="lbl_stealth_ip">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>40</y>
               <width>21</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>IP</string>
             </property>
            </widget>
            <widget class="QLabel" name="lbl_stealth_port">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>80</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_stealth_ip">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>40</y>
               <width>231</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="stealth_start_port">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>80</y>
               <width>91</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="stealth_end_port">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>80</y>
               <width>91</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>90</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>~</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_stealth">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>120</y>
               <width>80</width>
               <height>23</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(170, 170, 255);
	background-color: white;
	border: 2px solid rgb(170, 170, 255);
	border-radius: 5px;</string>
             </property>
             <property name="text">
              <string>스캔시작</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTabWidget" name="tabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>200</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>ping</string>
             </attribute>
             <widget class="QTextBrowser" name="output_ping">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>321</width>
                <height>131</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_ping_reset">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>150</y>
                <width>111</width>
                <height>23</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 190, 11, 255), stop:1 rgba(251, 86, 7, 255));
	border-radius: 20px;</string>
              </property>
              <property name="text">
               <string>현재 탭 리셋</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Stealth</string>
             </attribute>
             <widget class="QTextBrowser" name="output_stealth">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>321</width>
                <height>131</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_stealth_reset">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>150</y>
                <width>111</width>
                <height>23</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 190, 11, 255), stop:1 rgba(251, 86, 7, 255));
	border-radius: 20px;</string>
              </property>
              <property name="text">
               <string>현재 탭 리셋</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>SYN</string>
             </attribute>
             <widget class="QTextBrowser" name="output_syn">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>321</width>
                <height>131</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_syn_reset">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>150</y>
                <width>111</width>
                <height>23</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 190, 11, 255), stop:1 rgba(251, 86, 7, 255));
	border-radius: 20px;</string>
              </property>
              <property name="text">
               <string>현재 탭 리셋</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>get_Stealth()</slot>
 </slots>
</ui>
